pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <groupId>org.datadog.jenkins.plugins</groupId>
   <artifactId>datadog</artifactId>
-  <version>0.3.0-SNAPSHOT</version>
+  <version>0.3.1-SNAPSHOT</version>
```

Comment:
Nvm, solved in this cherry-pick PR: https://github.com/DataDog/jenkins-datadog-plugin/pull/28